### DISCUSSION TITLE
Finite parts of Loop Integral

### DISCUSSION THREAD

**Discussion by jasper7427** [FeynCalc/feyncalc#287] [Q&A]
Hi,
I am new to FeynCalc and am trying to use it to evaluate the LSZ correction to an outgoing quark. 
As such I am trying to calculate the self-energy diagram of an on-shell quark in the massless approximation.
This is the expression I should get:
![image](https://github.com/user-attachments/assets/bc08c146-0a4f-4ba4-b351-f0d0ca006733)

I have tried simply evaluating this with TID and PaXEvaluateUVIRSplit from FeynHelpers which gives me the correct terms in 1/EpsilonIR and 1/EpsilonUV, however this does not give me the finite parts of the expression. Is it possible to get these also?

Thanks in advance

**Comment by vsht** [comment]
Hi,

there's a QED renormalization example shipped with FeynHelpers that already covers that.

PaXEvaluateUVIRSplit always returns the full result, so there should be nothing missing. 

The massless approximation is tricky though, because there you get scaleless log-divergent loop integrals that nevertheless shouldn't be discarded to avoid mixing UV with IR poles.

The textbook treatment is not really generalizable to more complicated examples, so in real life one would rather use infrared rearrangement techniques.


Am 11. November 2024 12:38:00 UTC schrieb J7427 ***@***.***>:

**Reply by jasper7427** [reply]
Hi, 
Thank you for your answer.

Does PaXEvaluate handle expressions with D in them correctly? How am I meant to handle an expression of the form 

(2-D) * (some integral that will give IR and UV poles) 

such that the D is correctly assigned to EpsilonIR and EpsilonUV in the corresponding terms from the integral? 

Would it make sense to take keep the masses until the end and then set them to 0 to avoid the complications you mention?

Thank you

**Reply by vsht** [reply]
> (2-D) * (some integral that will give IR and UV poles)
That should work, provided that you don't use IBPs.

> Would it make sense to take keep the masses until the end and then set them to 0 to avoid the complications you mention? 
This is usually the safest way, but it complicates the calculation significantly.
